src/router.js

```diff
@@ -182,6 +182,15 @@ export default new Router({
             component: () =>
                 import(/*webpackChunkName:"account"*/  "./lib/components/pages/TenantNewEntity")
         },
+        {
```

Comment:
Lets get rid of this route by getting the entityId as a query parameter to the above route "/tenants/:clientId/entities/new"...

Reply:
I changed the name of the router for unique identification and passing entity parameters without using URL.